CONTRIBUTING.md

```diff
@@ -55,9 +55,8 @@ When you're done making changes, check that they pass linting, formatting, tests
 ```
 npm run lint
 npm run fmt
-npm run test
 npm run build
-npm run build:docs
+npm run doc
```

Comment:
Could you actually remove this line entirely since there's no npm `doc` script?

Reply:
Whoops, scratch that there is and I totally forgot about it 😂

You can still remove this line because `npm run doc` is (perhaps confusingly) a script used to generate API docs/bundled type definitions and is grouped in as part of `npm run build` –– so there's no need to rerun the `doc` script separately

Reply:
done :)